models/iaf_psc_exp.h

```diff
@@ -98,6 +98,25 @@ receptor_type 1, in contrast, is filtered through an exponential
 kernel with the time constant of the excitatory synapse,
 tau_syn_ex. For an example application, see [4]_.
 
+Conversion from PSC to PSP:
```

Comment:
I'm not so happy with notation here (e.g. ``v`` instead of ``V_m``, many new variables introduced but only used once). Moreover, the documentation does not actually show how to convert PSCs into PSPs. Could we replace this whole block with a reference to the postsynaptic_potential_to_current function instead?

https://github.com/nest/nest-simulator/blob/6f48c7f23f675c08cbb6d696bf382ab80d51ccda/pynest/examples/Potjans_2014/helpers.py#L65

Reply:
@jhnnsnk: just wanted to check that you intended to define the step function in reverse from the common definition, i.e. Theta(t) = 1 for t <= 0?

Reply:
@clinssen No, that looks like a typo. Thanks for spotting it! The `\leq` should be replaced by `\geq` in https://github.com/nest/nest-simulator/blob/6f48c7f23f675c08cbb6d696bf382ab80d51ccda/pynest/examples/Potjans_2014/helpers.py#L75  @sarakonradi Could this be corrected with this PR as well?

Reply:
> Could we replace this whole block with a reference to the postsynaptic_potential_to_current function instead?

I really like @clinssen 's suggestion to just use a reference to the function. In particular because it avoids duplication of text, but also because knowing about the conversion is not essential for the neuron model (it is just additional information). How about just adding a sentence like: "For conversion between postsynaptic potentials (PSPs) and PSCs, please refer to the function ..." ?

Reply:
> I really like @clinssen 's suggestion to just use a reference to the function. In particular because it avoids duplication of text, but also because knowing about the conversion is not essential for the neuron model (it is just additional information). How about just adding a sentence like: "For conversion between postsynaptic potentials (PSPs) and PSCs, please refer to the function ..." ?

Done. I agree with both of you that it would make sense to remove duplicate section in the neuron model, replacing it with a short reference to the function in ``helpers.py``.